src/renderer/db.ts

```diff
@@ -1,8 +1,25 @@
 import Dexie from 'dexie';
+import { keybindingsDefault } from './hooks/useKeybindings';
 
 export const db = new Dexie('query-master');
 
 db.version(1).stores({
```

Comment:
You can increase the version. You can think the version as database migration. https://dexie.org/docs/Dexie/Dexie.version()